Editor/CoreAPI/GameLiftComputeManager.cs

```diff
@@ -32,13 +27,14 @@ public GameLiftComputeManager(CoreApi coreApi, IAmazonGameLiftWrapper wrapper)
             var webSocketUrl = await RegisterCompute(computeName, fleetId, fleetLocation, ipAddress);
             if (webSocketUrl != null)
             {
-                var computeNameResponse = _coreApi.PutSetting(SettingsKeys.ComputeName, computeName);
-                var ipAddressResponse = _coreApi.PutSetting(SettingsKeys.IpAddress, ipAddress);
-                var webSocketResponse = _coreApi.PutSetting(SettingsKeys.WebSocketUrl, webSocketUrl);
+                _stateManager.SelectedProfile.ComputeName = computeName;
```

Comment:
Realized i'm lacking a bit of context to what this PR is intended to do, so I'll hold off on reviewing further until I speak to Jackson about this.